### DISCUSSION TITLE
Is it possible (and/or how easy is it) to run borg using the user home directory as the root repository?

### DISCUSSION THREAD

**Discussion by darkfeline** [emacscollective/borg#158] [Support] [answered]
Basically the title.  I use my home directory as the git repo root for all of my dotfiles, and I'm wondering how easy is it to get Borg setup without creating a submodule for `~/.config/emacs`.  Based on the docs, it seems like some things might be hard coded for `~/.config/emacs` but other things are configurable.

**Comment by tarsius** [comment, marked as answer]
That should be possible now.
